.nvmrc

```diff
@@ -0,0 +1 @@
+v18.12.0
```

Comment:
Add extra line please
Follow the prittier suggestions

Reply:
done